src/Core/CoreKernel.php

```diff
@@ -308,7 +308,7 @@ protected function redirectToInstaller()
         // Error if installer not available
         if (!file_exists($this->basePath . '/install.php')) {
             throw new HTTPResponse_Exception(
-                'SilverStripe Framework requires a $databaseConfig defined.',
+                'SilverStripe Framework requires a .env configuration file',
```

Comment:
maybe mention environment variables as an alternative as well?

Reply:
preferred for dev environments right?
I remember production environments are against using it

Reply:
No that was the old dotenv library; We don't use that anymore.